app/src/androidTest/java/com/example/weatherapp/data/repo/StorageRepositoryImplTest.kt

```diff
@@ -0,0 +1,65 @@
+package com.example.weatherapp.data.repo
+
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.example.weatherapp.domain.models.LocationMethod
+import org.junit.runner.RunWith
+import org.junit.Before
+import org.junit.Test
+import com.google.common.truth.Truth.assertThat
+import com.example.weatherapp.data.mappers.toData
+import com.example.weatherapp.domain.models.Language
+import com.example.weatherapp.domain.models.Units
+
+@RunWith(AndroidJUnit4::class)
+class StorageRepositoryImplTest {
+
+    lateinit var storageRepository: StorageRepositoryImpl
+
+    @Before
+    fun setup() {
+        storageRepository = StorageRepositoryImpl(ApplicationProvider.getApplicationContext())
+    }
+
+    @Test
```

Comment:
every test should have sections and appropriate fun title like

fun `GIVEN saved city name WHEN get city name THEN city name is correct`()
// GIVEN
// WHEN
// THEN